tools/accuracy_checker/configs/hbonet-1.0.yml

```diff
@@ -0,0 +1,66 @@
+models:
+  - name: hbonet-1.0
```

Comment:
@dkurt  why you have 3 models and only one config for AC?
I understand that they have difference only in path to model, but  it can confuse users that config for particular version is not supported

Reply:
I will add the rest of them, sure. I just wanted to test CI on a single one.